source/includes/_unified_display_report.md

```diff
@@ -0,0 +1,572 @@
+|`video_unmute`|Float|Yes|Number of Unmuted Video Views|
+|`video_pause`|Float|Yes|Number of Paused Video Views|
+|`video_resume`|Float|Yes|Number of Resumed Video Views|
+|`video_views`|Float|No|Number of Video Views|
+|`total_conversions`|Float|Yes|Number of Conversions|
+|`click_through_conversions`|Float|Yes|Number of Click-Through Conversions|
+|`view_through_conversions`|Float|Yes|Number of View-Through Conversions|
+|`video_played_to_25_pct`|Float|Yes|Number of Video Views Played to 25 percent|
+|`video_played_to_50_pct`|Float|Yes|Number of Video Views Played to 50 percent|
+|`video_played_to_75_pct`|Float|Yes|Number of Video Views Played to 75 percent|
+|`completed_views`|Float|No|Total Completed Views for Campaign|
+|`video_complete_rate`|Float|No|Total Video Completion Rate|
+|`view_thru_rate`|Float|No|Total View-Through Rate|
+
+<a name="udlineitemscreativesintervals"></a>
+#### Line Items Creatives Intervals Object
+| Field Name | Datatype | Nullable | Description |
+|---|---|---|---|
+|`start_date`|String|No|Start Date of Interval|
+|`impressions`|Int|No|Total Impressions for Interval|
+|`clicks`|Int|No|Total Clicks for Interval|
+|`ctr`|Float|No|Overall Click-through Rate of Interval|
+|`universal_interaction_rate`|Float|No|Overall Universal Interaction Rate for Interval|
+|`universal_interaction_time`|Float|No|Overall Universal Interaction Time for Interval|
+|`hover_rate`|Float|No|Hover Rate for Interval|
+|`attention_quality`|Float|No|Attention Quality for Interval|
+|`video_full_screen`|Float|Yes|Number of Full-Screen Video Views|
+|`video_mute`|Float|Yes|Number of Muted Video Views|
+|`video_unmute`|Float|Yes|Number of Unmuted Video Views|
+|`video_pause`|Float|Yes|Number of Paused Video Views|
+|`video_resume`|Float|Yes|Number of Resumed Video Views|
+|`video_views`|Float|Yes|Number of Video Views|
+|`total_conversions`|Float|Yes|Number of Conversions|
+|`click_through_conversions`|Float|Yes|Number of Click-Through Conversions|
+|`view_through_conversions`|Float|Yes|Number of View-Through Conversions|
+|`video_played_to_25_pct`|Float|Yes|Number of Video Views Played to 25 percent|
+|`video_played_to_50_pct`|Float|Yes|Number of Video Views Played to 50 percent|
+|`video_played_to_75_pct`|Float|Yes|Number of Video Views Played to 75 percent|
+|`completed_views`|Float|Yes|Number of Completed Views|
+|`video_complete_rate`|Float|Yes|Rate of Video Completion vs Partial Views|
+|`view_thru_rate`|Float|Yes|View-Through Rate|
+
+### Examples:
+
+> Retrieve data for a specific range of dates
+
+```
+curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
+https://api.reachlocalservices.com/client_reports/unified_display/USA_105569?start_date=2016-12-01&end_date=2016-12-31
+```
+
+> Retrieve data with intervals
+
+```
+curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
+https://api.reachlocalservices.com/client_reports/unified_display/USA_105569?start_date=2016-12-01&end_date=2016-12-31&interval_size=day
```

Comment:
Lets change USA_105569 to USA_123 to match the example response and to scrub client specific data from our documentation.

Reply:
I updated these to USA_123 instead to match the response and remove client specific data. I did want to know if you wanted me to do this across the documentation as a whole (since it appears there are other APIs that have account specific data). Or if you would prefer that be a future update in a separate PR to not delay this from going out.